package.json

```diff
@@ -21,7 +21,7 @@
     "docs": "nuxt docs",
     "lint": "eslint --ext .js,.ts,.vue",
     "prepack": "yarn build",
-    "dev:setup": "nuxt-module-build --stub && nuxi prepare playground",
+    "dev:setup": "nuxt-module-build --stub && nuxt prepare playground",
```

Comment:
`nuxi` is standard CLI since nuxt3. You can use it with nuxt 2 as well but usage is to avoid conflicting CLIs

Reply:
sorry that i didnt recognize the `nuxi` command.